caravel/assets/javascripts/SqlLab/main.css

```diff
@@ -238,14 +235,6 @@ div.tablePopover:hover {
 .ace_content {
     background-color: #f4f4f4;
 }
-.ws-el > .ws-el-controls {
```

Comment:
happy to discuss the tradeoffs here! i think a PR is a good place to discuss. 

my rationale is that these icons/actions are more discoverable if we don't hide them. 

if they are always visible, as the user starts using the product they will be more likely to see them and use them. if they only see them on hovering of the table name, it could be easy to forget where the ui is for these actions.

what is the advantage of hiding them here?